azerothcore/azerothcore-wotlk
# Fix(DB/Gameobject) Mining nodes inside of eachother

## Changes Proposed:
Moves the node that is inside of another to a more appriopiate location.


## Issues Addressed:
- Closes #5338
- Closes #5310



## SOURCE:

Source is original bug report.

## Tests Performed:

Works on Windows 10 

## How to Test the Changes:

Please follow the wiki to update your database, when done use `.go o 7327` &  `.go o 12487` and see the change.

## Known Issues and TODO List:

N/A


## Target Branch(es):







 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
